modules/core/src/main/scala/org/scalasteward/core/repoconfig/UpdatesConfig.scala

```diff
@@ -74,7 +74,7 @@ final case class UpdatesConfig(
 
 object UpdatesConfig {
   val defaultFileExtensions: Set[String] =
-    Set(".scala", ".sbt", ".sbt.shared", ".sc", ".yml", "pom.xml")
+    Set(".mill-version", ".scala", ".sbt", ".sbt.shared", ".sc", ".yml", "pom.xml")
```

Comment:
I was assuming alphabetical order, that's why I choose to first position for the insert. But now I see, `pom.xml` isn't in order.